plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/utils/TableViewerBuilder.java

```diff
@@ -50,13 +51,14 @@ public TableViewerBuilder(TableViewer viewer, Composite tableContainer) {
 		viewer.setComparer(new EqualityComparer());
 		this.tableLayout = new TableColumnLayout();
 		tableContainer.setLayout(tableLayout);
-		FontData fd = viewer.getTable().getFont().getFontData()[0];
+		normalFont = viewer.getTable().getFont();
+		FontData fd = normalFont.getFontData()[0];
 		fd.setStyle(SWT.ITALIC);
-		italic = new Font(null, fd);
+		italicFont = new Font(null, fd);
```

Comment:
I use JFaceResources.getFontRegistry().getItalic(JFaceResources.DEFAULT_FONT)

According to http://stackoverflow.com/questions/1830058/styling-a-swt-label-to-be-italic#comment42024249_18272783 you don't need to dispose it yourself